src/crypto/keys.js

```diff
@@ -58,3 +58,20 @@ export function getPrivateAndPublicKeyFromSecret(secret) {
 export function getKeys(secret) {
 	return getPrivateAndPublicKeyFromSecret(secret);
 }
+
+/**
+ * @method getAddressFromSecret
+ * @param secret
+ *
+ * @return keys object
+ */
+
+export function getAddressFromSecret(secret) {
```

Comment:
The name of this function is misleading. Maybe `getAddressAndPublicKeyFromSecret` or `getAccountFromSecret`? @Tosch110

Reply:
Good idea, I would prefer getAddressAndPublicKeyFromSecret to be more accurate